app.js

```diff
@@ -9,85 +9,134 @@ const weather = {};
 
 weather.temperature = {
     unit : "fahrenheit"
-}
+};
 
 // F to C CONVERSION
 function fahrenheitToCelsius(temperature){
     return (temperature - 32) * (5/9);
-}
+};
 
-// SELECTING ELEMENTS
-const iconElement = document.querySelector(".weather-icon");
-const temperatureElement = document.querySelector(".temperature-value p");
-const descriptionElement = document.querySelector(".temperature-description p");
-const locationElement = document.querySelector(".location");
+// SELECTING ELEMENTS FOR CURRENT WEATHER 
+const currentIconElement = document.querySelector(".current-weather-icon");
+const currentTemperatureElement = document.querySelector(".current-temperature-value p");
+const currentDescriptionElement = document.querySelector(".current-weather-description p");
+const currentLocationElement = document.querySelector(".current-location");
 const notificationElement = document.querySelector(".notification");
 
+// SELECTING ELEMENTS FOR 3 DAY FORECAST ///////////////////////////////////////////////////////
```

Comment:
So for all the day selectors. Could you do a loop and create an object (map) of all of them? Maybe search for ".day-weather-icon*" or some kind of fuzzy search.

Reply:
Oooo yeah I like this idea, I'm gonna read into while I'm here at my parents and start creating it